ci/scripts/release.sh

```diff
@@ -124,23 +124,26 @@ if [[ -n "${BUILDKITE_TAG}" ]]; then
 
     echo "--- Release upload risingwave asset"
     tar -czvf risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz risingwave
-    gh release upload "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz
+    gh release upload --clobber "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz
 
     echo "--- Release upload risingwave debug info"
     tar -czvf risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.dwp.tar.gz risingwave.dwp
-    gh release upload "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.dwp.tar.gz
+    gh release upload --clobber "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.dwp.tar.gz
 
     echo "--- Release upload risectl asset"
     tar -czvf risectl-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz risectl
-    gh release upload "${BUILDKITE_TAG}" risectl-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz
+    gh release upload --clobber "${BUILDKITE_TAG}" risectl-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz
 
-    echo "--- Release upload connector libs asset"
-    tar -czvf risingwave-connector-"${BUILDKITE_TAG}".tar.gz libs
-    gh release upload "${BUILDKITE_TAG}" risingwave-connector-"${BUILDKITE_TAG}".tar.gz
+    connector_assets=$(gh release view "${BUILDKITE_TAG}" --json assets --jq '.assets[] | select(.name | contains("risingwave-connector"))' | wc -l)
+    if [[ ${connector_assets} -eq 0 ]]; then
```

Comment:
Do we still need to check existence given `--clobber`?

Reply:
`--clobber` is for the rerun scenario. But we can avoid uploading twice to save the pipeline execution duration by this check.